lib/service/src/main/java/io/atlasmap/service/AtlasService.java

```diff
@@ -261,30 +271,47 @@ public Response removeMappingRequest(@ApiParam("Mapping ID") @PathParam("mapping
     }
 
     @DELETE
-    @Path("/mapping/RESET")
+    @Path("/mapping/RESET/{mappingId}")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = "Remove All Mappings", notes = "Remove all mapping files saved on the server")
```

Comment:
Need to change the @ApiOperation to indicate you're removing the specified mappingId - not all mappings.  Which
reminds me - is there a mechanism for remving all mappings with on backend REST invocation?  Needed for 'Reset All'
in the UI.

Reply:
> Need to change the @ApiOperation to indicate you're removing the specified mappingId - not all mappings

Yes definitely. 

> is there a mechanism for remving all mappings with on backend REST invocation? Needed for 'Reset All'
in the UI.

When i was thinking about this i thought about when might the user require to reset all mappings and the current only, but yes it can be provided at least as back-end functionality even if not used in the front-end.